src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -450,7 +450,7 @@ public static Callable<Object> callable(final PrivilegedAction<?> action) {
      * @return a callable object
      * @throws NullPointerException if action null
      */
-    public static Callable<Object> callable(final PrivilegedExceptionAction<?> action) {
+    public static Callable<@Nullable Object> callable(final PrivilegedExceptionAction<?> action) {
```

Comment:
Same comment as above.

Reply:
Done in both places (and import added).

Thanks. I'm not sure if I'm just very used to my world with no `@PolyNull` or if it just never occurred to me that the method would be declared to return `Callable<Object>` when it could have been declared to return `Callable<T>`!